### ISSUE TITLE
[Feature]: Hide Ads in code editor

### ISSUE THREAD

**Issue by johannesmutter** [wallabyjs/console-ninja#263]
### Describe the feature you'd like.

I'm OK with receiving occasional notifications to be reminded to upgrade to the PRO version.
But I find these ads that are displayed right next to my code a bit intrusive.

What's the reasoning behind displaying them there?

```
ced-1-TextEditorDecorationType152-54613a79-4 ced-1-TextEditorDecorationType152-4
```

![CleanShot 2024-02-02 at 02 24 33@2x](https://github.com/wallabyjs/console-ninja/assets/9489658/6d61dfa7-b221-4667-8fcb-34cdef677571)

**Comment by smcenlly** [comment]
The reason behind placing a discreet info icon with a hover effect, especially when using the `console.warn` method, is twofold:

1. **Educational:** Our primary intention is to provide immediate context and assistance by explaining why certain outputs might not appear as expected in the non-PRO version. This approach is meant to be informative and supportive, particularly for new users who may not be familiar with the limitations of our standard offering, or worse who think that Console Ninja is broken because it is not displaying a value.
2. **Awareness of PRO Features:** While this feature does inform users about the enhanced functionalities available in the PRO version, it's important to note that it's not the primary intent. Our goal is to subtly highlight the benefits of upgrading without disrupting your coding experience.

In light of your feedback, we are considering how to better display this information. We think we may be able to update it to only display if you hover over the icon itself instead of anywhere on the line.

**Comment by ArtemGovorov** [comment]
> We think we may be able to update it to only display if you hover over the icon itself instead of anywhere on the line.

We have changed the tip for Community feature set users to only show up if you hover over beyond the end of the line with your code (so in most of cases over the tip icon itself). The change was released in version v1.0.284.
